README.md

```diff
@@ -61,6 +61,8 @@ use: "file-loader?name=[name].[ext]&publicPath=assets/foo/&outputPath=app/images
 }
 ```
 
+`cssOutputPath` also should be used (together with `useRelativePath` property) to define the output path context.
```

Comment:
Merging them would be a breaking change, please leave it for the moment I need to think about how to handle it at best first 😛 it maybe better a separate PR in conjunction with `webpack-defaults` upgrade and another semver majors e.g the general paths unification like I mentioned before